src/gatsby-theme-carbon/components/LeftNav/ResourceLinks.js

```diff
@@ -2,17 +2,21 @@ import React from 'react';
 import ResourceLinks from 'gatsby-theme-carbon/src/components/LeftNav/ResourceLinks';
 
 const links = [
-  {
-    title: 'Github',
-    href: 'https://github.com/carbon-design-system/gatsby-theme-carbon',
-  },
-  {
-    title: 'Carbon Design System',
-    href: 'https://react.carbondesignsystem.com',
-  },
+    {
+        title: 'Github',
+        href: 'https://github.com/carbon-design-system/carbon-for-ibm-website',
+    },
+    {
+        title: 'IBM.com Library',
```

Comment:
@JennySanchez added the URL for IBM.com Library here. What I noticed though for Carbon Design System is that it links to their Storybook output and not their main site. Not sure which one we should point to, and if we should follow the same idea as them and point to our Storybook output on our end?